selftests/unit/test_runner_sysinfo.py

```diff
@@ -0,0 +1,55 @@
+import os
+import unittest
+
+from avocado.core.nrunner import Runnable
+from avocado.core.runners.sysinfo import SysinfoRunner
+from avocado.core.settings import settings
+
+
+class BasicTests(unittest.TestCase):
+    """Basic unit tests for the RequirementPackageRunner class"""
+
+    def in_message_path(self, messages, path, sysinfo_type='pre'):
+        path = os.path.join('sysinfo', sysinfo_type, path)
+        for message in messages:
+            if message.get('path', '') == path:
+                return True
+        return False
+
+    def test_pre(self):
+        kwargs = {'sysinfo_files': {'commands': ['uptime', 'dmidecode'],
```

Comment:
Unless I'm missing something, this sounds a bit strange to me:

The key is `sysinfo_files` but inside we do have `files` + `commands`.

Maybe:

```python
{'sysinfo': {'commands': ['uptime', 'dmidecode'],
             'files': ['/proc/version', '/proc/meminfo']}}
```

Reply:
sure, I can change that.